### PR TITLE
Read kamar list

### PR DESCRIPTION
**Pull request by saepasomba**
### Changes
- UI read kamar
- Filtering mechanism (Facilities based, availability based)
- Search mechanism

### Todo
- Integration with BE

### Preview
Updated header:
Screen Shot 2022-11-19 at 19 00 42
Updated kamar card:
Screen Shot 2022-11-19 at 19 00 53
Screen Shot 2022-11-17 at 22 58 13
Screen Shot 2022-11-17 at 22 58 27

### CODE REVIEW

File: src/components/cardKamar/CardKamar.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+import React from 'react'
+
+export default function CardKamar(props) {
+
+  const {
+    no_kamar,
+    wc_dalam,
+    ac,
+    springbed,
```
Comment (RadhiansyaZ) [inline src/components/cardKamar/CardKamar.js]: springBed

File: src/routes/kamar/ListKamar.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+import { FaBed, FaCheck, FaSearch, FaShower, FaTemperatureLow } from 'react-icons/fa'
+import { AiFillThunderbolt } from 'react-icons/ai'
+import { GiCancel } from 'react-icons/gi'
+
+const dummyData = [
+  {
+    no_kamar: 'No. 123',
+    wc_dalam: true,
+    ac: true,
+    springbed: true,
```
Comment (RadhiansyaZ) [inline src/routes/kamar/ListKamar.js]: dummy data dari `springbed` --> `springBed`

File: src/routes/kamar/ListKamar.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            <Button colorScheme='orangeChill'  {...(filters.includes('wc_dalam') ? null : {variant:'outline'})} onClick={() => filterBtnClicked('wc_dalam')} leftIcon={<FaShower />}>
+              Kamar mandi dalam
+            </Button>
+            <Button colorScheme='orangeChill' {...(filters.includes('ac') ? null : {variant:'outline'})} onClick={() => filterBtnClicked('ac')} leftIcon={<FaTemperatureLow />}>
+              AC
+            </Button>
+            <Button colorScheme='orangeChill' {...(filters.includes('listrik') ? null : {variant:'outline'})} onClick={() => filterBtnClicked('listrik')} leftIcon={<AiFillThunderbolt />}>
+              Listrik include
+            </Button>
+            <Button colorScheme='orangeChill' {...(filters.includes('springbed') ? null : {variant:'outline'})} onClick={() => filterBtnClicked('springbed')} leftIcon={<FaBed />}>
```
Comment (RadhiansyaZ) [inline src/routes/kamar/ListKamar.js]: springBed

**Review by RadhiansyaZ — COMMENTED** [review]
field di dummy data ada baiknya direvisi dikit sih E wkwkwk

**Review by luthfifahlevi — APPROVED** [review]

**Pull request merged**